src/unix/udp.c

```diff
@@ -167,13 +230,23 @@ static void uv__udp_recvmsg(uv_udp_t* handle) {
 
   do {
     buf = uv_buf_init(NULL, 0);
-    handle->alloc_cb((uv_handle_t*) handle, 64 * 1024, &buf);
+    handle->alloc_cb((uv_handle_t*) handle, UDP_DGRAM_MAXSIZE, &buf);
     if (buf.base == NULL || buf.len == 0) {
       handle->recv_cb(handle, UV_ENOBUFS, &buf, NULL, 0);
       return;
     }
     assert(buf.base != NULL);
 
+    uv_once(&once, uv__udp_mmsg_init);
+    if (uv__recvmmsg_avail) {
+      /* Returned space for more than 1 datagram, use it to receive
+       * multiple datagrams. */
+      if (buf.len >= 2 * UDP_DGRAM_MAXSIZE) {
+        uv__udp_recvmmsg(handle, &buf);
+        return;
```

Comment:
I'd allow multiple uv__udp_recvmmsg to be run - have uv__udp_recvmmsg return number of packets received and subtracting it from `count` until we get it to 0 (same as with regular recvmsg, to void loop startvation) - otherwise if we get a constant stream of packets we have to rearm event loop every time.

Reply:
Sure, that sounds reasonable. Are you going to implement it? :-)